lib/OnboardingMessageProvider.jsm

```diff
@@ -343,6 +343,18 @@ const ONBOARDING_MESSAGES = () => [
     targeting: "trailheadTriplet == 'payoff'",
     trigger: { id: "showOnboarding" },
   },
+  {
+    id: "TRAILHEAD_OVERLAY_1",
+    template: "trailhead_overlay",
+    targeting: "trailheadInterrupt == 'control_trailhead'",
```

Comment:
need a better name for this interrupt, these are variants of overlay seen on first run take over.  Messaging shows both sync and triplet cards so may be rename to 'sync_cards_overlay'

Reply:
How about `full_page`? that is pretty descriptive